base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -78,16 +79,28 @@ public VideoStreamingParameters() {
         format = new VideoStreamingFormat();
         format.setProtocol(DEFAULT_PROTOCOL);
         format.setCodec(DEFAULT_CODEC);
+        stableFrameRate = true;
     }
 
+    /**
+     * new constructor of VideoStreamingParameters, which now has stableFrameRate param.
+     * @param displayDensity
+     * @param frameRate
+     * @param bitrate
+     * @param interval
+     * @param resolution
+     * @param format
+     * @param stableFrameRate
+     */
     public VideoStreamingParameters(int displayDensity, int frameRate, int bitrate, int interval,
-                                    ImageResolution resolution, VideoStreamingFormat format) {
-        this.displayDensity = displayDensity;
-        this.frameRate = frameRate;
-        this.bitrate = bitrate;
-        this.interval = interval;
-        this.resolution = resolution;
-        this.format = format;
+                                    ImageResolution resolution, VideoStreamingFormat format, boolean stableFrameRate){
+	    this.displayDensity = displayDensity;
+	    this.frameRate = frameRate;
+	    this.bitrate = bitrate;
+	    this.interval = interval;
+	    this.resolution = resolution;
+	    this.format = format;
+	    this.stableFrameRate = stableFrameRate;
```

Comment:
Hello @shiniwat, This would be a breaking change, please deprecate the old constructor and assign `this.stableFrameRate = true` and then create the new constructor with `stableFrameRate` as a parameter

Reply:
Thanks for the review. The ctor has been updated as you suggested.